li-utils/src/main/java/com/linkedin/common/urn/MlFeatureUrn.java

```diff
@@ -0,0 +1,27 @@
+package com.linkedin.common.urn;
+
+public final class MlFeatureUrn extends Urn {
```

Comment:
I think adding namespace to the feature urn is necessary here to identify features with the same name and same type but used in different context.

 Also I don't think the type is needed in the urn. Assuming the combination of (namespace,name) would be unique for a feature, we don't need the type in the urn. It can be an aspect.

Reply:
Here are the list of changes i am making relating to this comment:
 - Added `MlFeature` as a top level entity with `MlFeatureSnapshot` and `MlFeatureAspect` 
 - Removed `dataType` as part of the `MlFeatureUrn` and instead added a string `featureNamespace`